### DISCUSSION TITLE
Change the colour of site messages if they are not errors

### DISCUSSION THREAD

**Discussion by TeddyWestside22** [RetroAchievements/RAWeb#1020] [Ideas]
Currently, all site messages display as red, regardless of their nature. I propose that messages that aren't errors, such as "Performed OK!" should be green. It is minor, but red is typically associated with failure/errors, so it would be more logical to have green for success, red for failure.

Example here:

![image](https://user-images.githubusercontent.com/32810074/172087056-b38137be-83f3-43a6-ab03-1265952f4813.png)

Other mock-ups:
![image](https://user-images.githubusercontent.com/32810074/172114311-ce38317d-416b-4d71-9d80-351911ae81c1.png)

**Comment by Jamiras** [comment]
implemented in #1030

**Reply by TeddyWestside22** [reply]
Perfect! Thank you for your work in implementing this.
